File: Leetcode-Solutions/Arrays/moveZeroes.py
class Solution(object):
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
        try:
            index=nums.index(0)
        except ValueError:
            return
        no_zeros=[i for i in nums if i!=0]
        zeros=[0] * (len(nums)-len(no_zeros))
        nums[:]=no_zeros+zeros
      

File: Leetcode-Solutions/Arrays/moveZeroes.js

/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var moveZeroes = function(nums) {
    let no_zeroes=nums.filter(x=>x!==0)
    let zeroes=Array(nums.length-no_zeroes.length).fill(0)
    nums.splice(0,nums.length,...no_zeroes,...zeroes)

};
